scripts/services/CreateConflationTypeGroups.py

```diff
@@ -42,12 +42,21 @@ def toInput(optionType):
 def toLabel(conflateType):
     return 'Poi to Polygon' if conflateType == 'PoiPolygon' else conflateType;
 
+def getDefaultValue(templateKey):
+    defaultValueKey =  ''.join( c for c in templateKey if c not in '${}').replace('.', ' ').title().replace(' ', '')
+    defaultValue = configOptions[defaultValueKey]['default']
+    if '${' in defaultValue:
+        getDefaultValue(defaultValue)
+    else:
+        return defaultValue;
+
  # merges matches between ConfigOptions.json and conflationTypesMap.json
  # on command key back into conflationTypesMap. Outputs as ConflationTypesGroups.json
 def createUiJSON(groups, options):
     try:
         uiGroupsArray = []
         configMembers = [{'key': key, 'member': member} for key, member in options.items()]
+        configDefault = [{'default': default } for default in options.items()]
```

Comment:
This variable does not seem to be used anywhere. maybe we can whack it?

Reply:
@maxgrossman removed